### PR TITLE
improve typography

### PR DESCRIPTION
**Pull request by vladh**
@chadwhitacre 

(Also cleaned up some CSS.)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,13 +4,28 @@
...
+        font-display: swap;
+        src: url(/fonts/ComicRelief-Bold.woff2) format('woff2');
+      }
+
       body {
-        background: #000000 url("bg.webp");
+        background: #000000 url(images/bg.webp);
         color: #00ff00;
-        font-family: "Comic Sans MS", cursive;
+        font-family: 'Comic Relief', 'Comic Sans MS', cursive;
```
Comment (chadwhitacre) [inline index.html]: What's the reason to prefer Comic Relief here?

Comment (vladh) [reply index.html]: A lot of machines, mine included, don't have Comic Sans MS, and the cursive fallback font is very hard to read. We can bundle Comic Relief to ensure the page looks the same to all users.

Comment (vladh) [reply index.html]: This is how the page looks for me — some words are hard to make out:

Comment (chadwhitacre) [reply index.html]: I noticed this on my phone this morning. I think we should prefer Comic Sans for the authentic Geocities experience and use Comic Relief as the fallback. Wdyt?

Comment (vladh) [reply index.html]: Done!

**Pull request merged**